tests/compile/tsick.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Lightweight conversion from tsc to Closure Compiler.
+ */
+'use strict';
+
+const fs = require('fs');
+
+const DIR = '../../build/src/core/';
+
+function spider(dir) {
+  const data = fs.readdirSync(dir, {withFileTypes: true});
```

Comment:
I think `entries` or `dirEntries` (and later `entry` or `dirEntry`) would be a little more specific.